lib/elixometer.ex

```diff
@@ -369,11 +369,13 @@ defmodule Elixometer do
       reporter = cfg[:reporter]
       interval = cfg[:update_frequency]
       subscribe_options = cfg[:subscribe_options] || []
+      datapoints_blacklist = cfg[:datapoints_blacklist] || []
 
       if reporter do
         metric_name
         |> :exometer.info
         |> Keyword.get(:datapoints)
+        |> Enum.filter(fn(datapoint) -> not Enum.member?(datapoints_blacklist, datapoint) end) 
```

Comment:
`Enum.reject/2` is a more natural fit here:

```elixir
Enum.reject(&Enum.member?(datapoints_blacklist, &1))
```

Reply:
I was looking for this but missed the `reject`, was looking for `difference`, or `intersect` ^^. Added it in, thanks!